data-reconciliation-app/Makefile

```diff
@@ -10,20 +10,29 @@ build: ## 🔨 Build the Application
 	@echo -e "\e[34m$@\e[0m" || true
 	@npm run build
 
-build-virtual: build ## 📦 Build Virtual container image from Dockerfile
+run-env-setup: create-jwt-configs build  ## 🔨 Create the initialization config files if not exists, and Build the Application
```

Comment:
I've gone around the houses on this and tried to understand WHAT run-env-setup actually means as the name doesn't really tell me much. I have the sense that you might make it cleaner if your `make build` step just also called `npm run create-jwt-configs`. This is then more like the set_js_app.json file that is created as part of `make build`

Reply:
nit: Unsure why it's "configs" plural

Reply:
Sorted